src/sql/workbench/services/notebook/browser/models/modelInterfaces.ts

```diff
@@ -548,6 +548,7 @@ export interface ICellModel {
 	 * Returns the name of the attachment added to metadata.
 	 */
 	addAttachment(mimeType: string, base64Encoding: string, name: string): string;
+	lastEditMode: string;
```

Comment:
The public interface for this should probably be readonly. 

But also - does it need to be in the public interface? This seems like a purely internal thing currently.

Reply:
good point, am not using it outside, removed it and made it private. Thanks!